### PR TITLE
update tracing init

### PR DESCRIPTION
**Pull request by abhilashshetty04**

### CODE REVIEW

File: call-home/src/bin/stats/main.rs [hunk 1]
```diff
@@ -154,10 +154,11 @@ async fn main() -> errors::Result<()> {
 }
 
 /// Initialize logging components -- tracing.
-pub(crate) fn init_logging(args: &Cli) {
+pub(crate) fn init_logging(_args: &Cli) {
     let tags = default_tracing_tags(raw_version_str(), env!("CARGO_PKG_VERSION"));
-
-    init_tracing("stats", tags, args.jaeger.clone());
+    TracingTelemetry::builder()
```
Comment (tiagolobocastro) [inline call-home/src/bin/stats/main.rs:168]: missing jeager and format args cli

File: metrics-exporter/src/bin/io_engine/main.rs [hunk 2]
```diff
@@ -62,9 +63,10 @@ pub(crate) fn grpc_client<'a>() -> &'a GrpcClient {
 async fn main() -> Result<(), ExporterError> {
     let args = Cli::args();
     utils::print_package_info!();
-
-    utils::tracing_telemetry::init_tracing("metrics-exporter-io_engine", vec![], None);
-
+    utils::tracing_telemetry::TracingTelemetry::builder()
+        .with_writer(FmtLayer::Stdout)
+        .with_style(FmtStyle::Pretty)
```
Comment (tiagolobocastro) [inline metrics-exporter/src/bin/io_engine/main.rs]: Should be specified from cli args as well?

File: k8s/upgrade/src/bin/upgrade-job/main.rs [hunk 3]
```diff
@@ -41,7 +41,9 @@ async fn main() -> Result<()> {
 fn init_logging() {
     let tags = default_tracing_tags(raw_version_str(), env!("CARGO_PKG_VERSION"));
 
-    init_tracing("upgrade-job", tags, None);
+    TracingTelemetry::builder()
+        .with_tracing_tags(tags)
```
Comment (tiagolobocastro) [inline k8s/upgrade/src/bin/upgrade-job/main.rs:45]: missing format things from cli args? Similar to your changes on control-plane repo

**Review by Abhinandan-Purkait — APPROVED** [review]

**Review by tiagolobocastro — COMMENTED** [review]
Please squash your commits to the last one, this is not a cherry-pick

**Pull request closed without merging**